diff --git a/config.example.toml b/config.example.toml
index e66e95c..a065bf3 100644
--- a/config.example.toml
+++ b/config.example.toml
@@ -5,7 +5,7 @@
 
     # # Domain which the instance is hosted. By default it is unset.
     # # This is primary used to generate links to the instance
-    # domain =
+    domain = "https://priviblur.fly.dev"
 
     # Enables secure cookies and forces all links
     # to priviblur to use the `https://` scheme
@@ -77,4 +77,4 @@
 
 # [misc]
     # # Enable sanic's dev mode
-    # dev_mode = false
\ No newline at end of file
+    # dev_mode = false
diff --git a/docker/Dockerfile b/docker/Dockerfile
index f748a4a..0755149 100644
--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -5,6 +5,7 @@ COPY ./.git/ ./.git/
 COPY ./requirements.txt ./requirements.txt
 COPY ./src/ ./src/
 COPY ./assets/ ./assets/
+COPY ./config.example.toml ./config.toml
 COPY ./locales/ ./locales/
 
 RUN apk add --no-cache git python3 py3-setuptools tini && \
